.github/workflows/sn_pr_tests.yml

```
@@ -297,11 +297,15 @@ jobs:
         run: ./target/release/log_cmds_inspector $HOME/.safe/node/local-test-network
         if: matrix.os != 'windows-latest'
 
+      - name: Tar log files
+        run: find ~/.safe/node/local-test-network -iname '*.log*' | tar -zcvf log_files.tar.gz --files-from -
```

I noticed this error line in the ci run:
```
tar: /home/ec2-user/.safe/node/local-test-network/sn-node-4/sn_node.log: file changed as we read it
```
perhaps kill off any running nodes before tarring the logs? i.e.

```suggestion
        run: killall sn_node; find ~/.safe/node/local-test-network -iname '*.log*' | tar -zcvf log_files.tar.gz --files-from -
```